pwa/src/services/dateTime.tsx

```diff
@@ -19,3 +19,19 @@ export const dateTime = (language: TLanguageCodes, dateTime: string) => {
     </div>
   );
 };
+
+export const dateTimeTag = (language: TLanguageCodes, dateTime: string) => {
+  if (!dateTime) return;
+
+  let _dateTime = Date.parse(dateTime);
```

Comment:
I don't think we actually need this `let` declaration.

`const newDate = new Date(Date.parse(dateTime);` might be better.